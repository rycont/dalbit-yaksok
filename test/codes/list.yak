소수: [2, 3, 5, 7, 11, 13, 17, 19]
소수 보여주기
