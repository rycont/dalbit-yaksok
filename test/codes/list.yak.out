[떡볶이, 라면, 순대, 튀김, 어묵, 슬러시]
떡볶이
라면
순대
튀김
어묵
슬러시
[떡볶이, 순대, 어묵]
먹고 싶다 튀김
[1, 2, [3, 4]]
홍길동1
[11, 0, 야옹야옹야옹, 2, 2]
